dockerd-entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
this is the default entrypoint file, from here:
https://github.com/aws/aws-codebuild-docker-images/blob/master/ubuntu/standard/6.0/dockerd-entrypoint.sh